src/api/models.py

```diff
@@ -1,12 +1,18 @@
 from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime
 
 db = SQLAlchemy()
 
 class User(db.Model):
+    __tablename__ = 'user'
     id = db.Column(db.Integer, primary_key=True)
-    email = db.Column(db.String(120), unique=True, nullable=False)
```

Comment:
I thought we were only using id, email, and password for our user model?

Reply:
Sure I just commented them out for future additions. Everything should be up to date now.

Reply:
Sweeeet